### DISCUSSION TITLE
ED060SCL support

### DISCUSSION THREAD

**Discussion by dgudim** [vroland/epdiy#309] [Q&A]
Hello. I have a ED060SCL. From the supported devices list it seems this one is not supported. But it has 39 pins, is there any chance it will work?

**Comment by martinberlin** [comment]
Sure it should work but at least latest v7 does not have this connector.
So you will have to clone the hardware and edit the PCB yourself. I would highly recommend to check for [Inkster](https://github.com/mcer12/Inkster-ESP32) board designed by @mcer12 (it has this connector)
But if you want to go for latest and fastest board I would always recommend v7 with esp32s3 and just update it with the connector you desire (for example removing one of the other connectors)
Alternatively just get one of the other supported displays.
Moving this to Discussions section since it’s not really an issue or bug with the component

**Comment by dgudim** [comment]
> Sure it works

Cool

> update it with the connector you desire

I don't think i can pull that off, i think I'll just use v6

**Comment by vroland** [comment]
It shouldn't be too complex actually, since you can just copy the part from V6 and one of the connectors from V7, mirror it and connect up the right signals ;) If you do, feel free to open a PR on the hardware repo as well :)
